lib/killbill_client/models/combo_transaction.rb

```diff
@@ -34,14 +36,18 @@ def combo_payment(user, reason, comment, options, refresh_options = nil)
                                                 }.merge(options)
         rescue KillBillClient::API::ResponseError => error
           response = error.response
-          if response.header['location']
+          if follow_location && response.header['location']
             created_transaction = ComboTransaction.new
             created_transaction.uri = response.header['location']
           else
             raise error
           end
         end
-        created_transaction.refresh(refresh_options || options, Payment)
+
+        if follow_location
+          return created_transaction.refresh(refresh_options || options, Payment)
+        end
+        created_transaction
```

Comment:
Could we return a `Payment` instance with just the `uri` attribute set?

Otherwise, if the user didn't specify a key, there is no easy way to retrieve later on the payment, is there?